package.json

```diff
@@ -17,7 +17,7 @@
 	],
 	"types": "./@types/index.d.ts",
 	"engines": {
-		"node": ">=10.16"
+		"node": ">=10.17"
```

Comment:
Why was the node engine bumped up a version?

Reply:
To allow use of `Readable.from`. It will be also required for new version of `fetch-blob` and #603 and #210.